src/Core/Common/ImageRegionOverlap/Code.cxx

```diff
@@ -136,15 +136,10 @@ main(int argc, char * argv[])
     ++itC;
   }
 
-  // Writer
-  using FileWriterType = itk::ImageFileWriter<RGBImageType>;
-  FileWriterType::Pointer writer = FileWriterType::New();
-  writer->SetFileName(out_file_name);
-  writer->SetInput(image);
-
+  // Write image
```

Comment:
These kinds of comments should now be mostly removed.

Reply:
Done